pkg/storaged/lvol-tabs.jsx

```diff
@@ -110,6 +114,14 @@ function lvol_and_fsys_resize(client, lvol, size, offline, passphrase) {
                 return Promise.reject(_("VDO backing devices can not be made smaller"));
             else
                 return Promise.resolve();
+        } else if (stratis_bdev) {
+            if (size - crypto_overhead > Number(stratis_bdev.TotalPhysicalSize)) {
+                const pool = client.stratis_pools[stratis_bdev.Pool];
+                return pool.GrowPhysicalDevice(stratis_bdev.Uuid).then(std_reply);
+            } else if (size - crypto_overhead < Number(stratis_bdev.TotalPhysicalSize))
+                return Promise.reject(_("Stratis blockdevs can not be made smaller"));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230823-090550-a4293775-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)